src/NServiceBus.Msmq.AcceptanceTests/NServiceBus.Msmq.AcceptanceTests.csproj

```diff
@@ -89,6 +89,9 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
@danielmarbach sorry for the spam, turns out this was a VS2012(!) behavior that marked the project as a test project. It's no longer needed in VS2013+ https://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for